src/dnmd/query.c

```diff
@@ -933,3 +969,14 @@ bool md_find_cursor_of_range_element(mdcursor_t element, mdcursor_t* cursor)
         return false;
     return find_range_element(element, cursor);
 }
+
+mdcursor_t md_resolve_indirect_cursor(mdcursor_t c)
+{
+    if (table_is_indirect_table(CursorTable(&c)->table_id))
+    {
+        mdcursor_t pointed_to = { 0 };
+        (void)md_get_column_value_as_cursor(c, index_to_col(0, ExtractTable(CursorTable(&c)->table_id)), 1, &pointed_to);
```

Comment:
Is there any reason to think that a failure here is valid? If not, we should at least assert the result as "true".